kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -350,26 +351,26 @@ def timesince(d, now=None):
     chunks = [
         (
             60 * 60 * 24 * 365,
-            lambda n: ungettext("%(number)d year ago", "%(number)d years ago", n),
+            lambda n: ngettext("%(number)d year ago", "%(number)d years ago", n),
```

Comment:
note: when we used once `ngettext` in prod, it was failing with Django 2.x

Reply:
It appears to be the only option now - is there a way I should be checking for this failure outside of the unit tests? It appears to pass at the moment.  ?